opacclient/opacapp/src/main/assets/bibs/Mils.json

```diff
@@ -0,0 +1,12 @@
+{
+    "state": "Tirol",
+    "account_supported": false,
+    "title": "Bibliothek Mils",
+    "city": "Mils",
+    "country": "\u00d6sterreich",
+    "api": "web-opac.at",
+    "geo": [47.2883084, 11.5314953],
+    "data": {
+        "baseurl": "http://www.buecherei-mils.webopac.at"
+    }
+}
```

Comment:
This library is missing the `information` attribute and therefore the app does not display it